src/js/rise-cache.js

```diff
@@ -4,7 +4,7 @@ RiseVision.Common = RiseVision.Common || {};
 RiseVision.Common.RiseCache = (function () {
   "use strict";
 
-  var BASE_CACHE_URL = "http://localhost:9494/";
+  var BASE_CACHE_URL = "https://localhost:9494/";
```

Comment:
@rodrigopavezi Did the RC v1 allow for https? If not, won't this always fail for Widgets running on v1 or v2 players?